relion/protocols/protocol_base.py

```diff
@@ -991,7 +993,14 @@ def _setComputeArgs(self, args):
         args['--pool'] = self.pooledParticles.get()
 
         if self.doGpu:
-            args['--gpu'] = self.gpusToUse.get()
+            gpuStr = self.gpusToUse.get().strip()
```

Comment:
I didn't have any problems running relion without quotes, but why not

Reply:
I can't remember exactly the case, but Dustin had some issue, not  sure if using :  or spaces